lib/chef/resource/windows_task.rb

```diff
@@ -44,22 +44,22 @@ def initialize(name, run_context = nil)
       property :interactive_enabled, [TrueClass, FalseClass], default: false
       property :frequency_modifier, [Integer, String], default: 1
       property :frequency, equal_to: [:minute,
-                                       :hourly,
-                                       :daily,
-                                       :weekly,
-                                       :monthly,
-                                       :once,
-                                       :on_logon,
-                                       :onstart,
-                                       :on_idle,
-                                       :none], default: :hourly
+                                      :hourly,
+                                      :daily,
+                                      :weekly,
+                                      :monthly,
+                                      :once,
+                                      :on_logon,
+                                      :onstart,
+                                      :on_idle,
+                                      :none], default: :hourly
       property :start_day, String
       property :start_time, String
       property :day, [String, Integer]
       property :months, String
       property :idle_time, Integer
-      property :random_delay, String
-      property :execution_time_limit, String
+      property :random_delay, [String, Integer]
+      property :execution_time_limit, [String, Integer], default: "PT72H"
```

Comment:
comment on what a `PT72H` is when it's at home might be nice (or a link to some docs)

Reply:
Ah, I see from Google it's an ISO8601 duration. It would be good to document that that's what we're expecting here.

Reply:
I'll add a comment for that. The original block that injected the value had a comment